metastore/src/java/org/apache/hadoop/hive/metastore/HiveClientCache.java

```diff
@@ -107,7 +107,7 @@ public HiveClientCache(final int timeout) {
   /**
    * @param timeout the length of time in seconds after a client is created that it should be automatically removed
    */
-  private HiveClientCache(final int timeout, final int initialCapacity, final int maxCapacity, final boolean enableStats) {
+  protected HiveClientCache(final int timeout, final int initialCapacity, final int maxCapacity, final boolean enableStats) {
```

Comment:
Is it simpler to make this public and directly instantiate HiveClientCache in HCatUtil?

Reply:
WDYM, it is instantiated directly in HCatUtil, see https://github.com/apache/hive/pull/5924/files#diff-70dc922fa5f53c56617a8ac9684b41630cd9aa1b673504c7cf29e2f5798a81acR573-R582